src/test/java/com/washingtonpost/legacy/README.md

```diff
@@ -1,4 +1,4 @@
-# Examples of current story content
-This directory holds examples of all the content that goes in to a current WashingtonPost.com story or video
+# Examples of current collection content
```

Comment:
Can you roll back your find & replace for this file?  It's talking about legacy story (small 's', the concept, not big 'S', the class)